kivy/core/window/__init__.py

```diff
@@ -1835,7 +1844,8 @@ def configure_keyboards(self):
         self.bind(
             on_key_down=sk._on_window_key_down,
             on_key_up=sk._on_window_key_up,
-            on_textinput=sk._on_window_textinput)
+            on_textinput=sk._on_window_textinput,
+            on_textedit=sk._on_window_textedit)
```

Comment:
```suggestion
            on_textedit=sk._on_window_textedit,
        )
```
will make future edits cleaner